src/generators.cpp

```diff
@@ -207,7 +207,7 @@ std::string to_string(DeviceType device_type) {
     case DeviceType::DML:
       return "DirectML";
     case DeviceType::WEBGPU:
-      return "WebGpu";
+      return "WebGPU";
```

Comment:
Amusing, these ones are just for an error message and the 'GetDeviceType' method on the OgaModel. The 'QnnWithSharedMemory' one implies these are the memory type vs the device type. None are consistent here really.